packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -16731,6 +16731,34 @@ void main() {
     final EditableTextState state = tester.state<EditableTextState>(find.byType(EditableText));
     expect(state.renderEditable.cursorColor, cursorColor.withOpacity(opacity));
   });
+
+  testWidgets('should notify on size change', (WidgetTester tester) async {
+    int notifyCount = 0;
+    await tester.pumpWidget(MaterialApp(
+      home: Scaffold(
+        body: NotificationListener<SizeChangedLayoutNotification>(
+          onNotification: (SizeChangedLayoutNotification notification) {
+            notifyCount += 1;
+            return false;
+          },
+          child: EditableText(
+            backgroundCursorColor: Colors.grey,
+            cursorColor: Colors.grey,
+            controller: controller,
+            focusNode: focusNode,
+            maxLines: 3,
+            minLines: 1,
+            style: textStyle,
+          ),
+        ),
+      ),
+    ),);
+
+    await tester.enterText(find.byType(EditableText), '\n');
```

Comment:
Before this line, can you expect that the count is zero? Just to be clear that the notification happens due to the entered text and not during initial build.